benchmark_runner/common/oc/oc.py

```diff
@@ -242,6 +242,61 @@ def get_prom_token(self):
         long_uuid = self.run(f"oc -n openshift-monitoring sa get-token prometheus-k8s")
         return long_uuid
 
+    @typechecked
+    def exec(self, command: str, pod_name: str, namespace: str=''):
+        """
+        oc exec a command and return the answer
+        :return:
+        """
+        try:
+            if namespace != '':
+                namespace = f'-n {namespace}'
+            return self.run(f'oc exec {namespace} {pod_name} -- {command}')
+        except Exception as err:
+            raise ExecFailed(command, pod_name, err)
+
+    @typechecked
+    def terminate_pod_sync(self, pod_name: str,
+                           namespace: str = environment_variables.environment_variables_dict['namespace'],
+                           timeout: int = int(environment_variables.environment_variables_dict['timeout'])):
+
+        """
+        Delete a pod with no associated YAML file
+        """
+        if self._is_pod_exist(pod_name, namespace):
+            try:
+                if namespace != '':
+                    namespace = f'-n {namespace}'
+                self.run(f'oc delete pod {namespace} {pod_name} timeout={timeout}')
+            except Exception as err:
+                raise PodTerminateTimeout(pod_name)
+
+    @typechecked
+    @logger_time_stamp
+    def wait_for_pod_ready(self, pod_name: str,
```

Comment:
Why we need another method, we already have 'wait_for_ready'

Reply:
We don't.  I missed it.  My bad.

Reply:
Well, actually I'm going to need to modify it, because I can't use the `label`.

Reply:
It looks like wait_for_ready is just too different from my use case; it's intended to watch for our pods, not for arbitrary system pods.